sdks/python/setup.py

```diff
@@ -222,6 +222,17 @@ def get_version():
     'ipython>=5.8.0,<8',
     'timeloop>=1.0.2,<2',
 ]
+
+INTERACTIVE_BEAM_TEST = [
```

Comment:
This is a quite a large addition for the value we are getting.

And,
- should this be integrated with tox?
- jenkins?
- which platforms does this test work in?

Reply:
Fair. Please add this information to the wiki. (https://cwiki.apache.org/confluence/display/BEAM/Python+Tips or a new notebook tips page)

Reply:
Will update the readme in another PR once this is submitted